wp-content/themes/core/assets/css/src/admin/acf/all.pcss

```diff
@@ -0,0 +1,37 @@
+/* Image Select Field Mods */
```

Comment:
Additionally, it would be helpful to add a proper comment header and comment blocks to these pcss particals. Check out the other files for a reference to our comments formatting.